app/controllers/foreman_deployments/api/v2/stacks_controller.rb

```diff
@@ -20,6 +20,18 @@ def create
           process_response @stack.save
         end
 
+        api :PUT, '/stacks/:id/', N_('Update imported stack')
+        param_group :stack, :as => :update
+        param :id, :identifier, :required => true
+        def update
+          if @stack.configurations.empty?
```

Comment:
Once the stack is configured, there is no way to update it. We should add an option to remove configurations too.